test/tests/components/ws-manager/content_test.go

```diff
@@ -0,0 +1,228 @@
+// Copyright (c) 2020 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package wsmanager
+
+import (
+	"context"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"sigs.k8s.io/e2e-framework/pkg/envconf"
+	"sigs.k8s.io/e2e-framework/pkg/features"
+
+	content_service_api "github.com/gitpod-io/gitpod/content-service/api"
+	agent "github.com/gitpod-io/gitpod/test/pkg/agent/workspace/api"
+	"github.com/gitpod-io/gitpod/test/pkg/integration"
+	test_context "github.com/gitpod-io/gitpod/test/pkg/integration/context"
+	wsapi "github.com/gitpod-io/gitpod/ws-manager/api"
+)
+
+// TestBackup tests a basic start/modify/restart cycle
+func TestBackup(t *testing.T) {
+	backup := features.New("backup").
+		WithLabel("component", "ws-manager").
+		Setup(func(ctx context.Context, t *testing.T, cfg *envconf.Config) context.Context {
+			api := integration.NewComponentAPI(ctx, cfg.Namespace(), cfg.Client())
+			return test_context.SetComponentAPI(ctx, api)
+		}).
+		Assess("it should start a workspace, create and file and successfully create a backup", func(ctx context.Context, t *testing.T, cfg *envconf.Config) context.Context {
```

Comment:
/very minor nit

"create and file" => "create a file"

Reply:
fixed